test/common/http/http1/codec_impl_test.cc

```diff
@@ -3317,43 +3255,43 @@ TEST_P(Http1ClientConnectionImplTest, LowWatermarkDuringClose) {
 TEST_P(Http1ServerConnectionImplTest, LargeTrailersRejected) {
   // Default limit of 60 KiB
   std::string long_string = "big: " + std::string(60 * 1024, 'q') + "\r\n\r\n\r\n";
-  testTrailersExceedLimit(long_string, "http/1.1 protocol error: trailers size exceeds limit",
+  testTrailersExceedLimit(long_string, "http/1.1 protocol error: trailers size exceeds limit", true,
```

Comment:
Probably all of these false/true could do with `/* fieldname = */` comments to make the test readable.